CONTRIBUTING.md

```diff
@@ -89,3 +89,83 @@ Reference [pre-commit's installation instructions](https://pre-commit.com/#insta
+
+```
+CONTAINER ID   IMAGE                          COMMAND                  CREATED             STATUS                       PORTS                                 NAMES
+e3939c800021   codex_houston                  "/docker-entrypoint.…"   About an hour ago   Up About an hour             0.0.0.0:83->5000/tcp                  codex_houston_1
+5c0b2ab64bdc   dpage/pgadmin4                 "/entrypoint.sh"         22 hours ago        Up About an hour             443/tcp, 0.0.0.0:8000->80/tcp         codex_pgadmin_1
+4761da5d9168   redis:latest                   "docker-entrypoint.s…"   22 hours ago        Up About an hour             6379/tcp                              codex_redis_1
+9560f26f0227   wildme/edm:latest              "/usr/local/tomcat/b…"   22 hours ago        Up About an hour             0.0.0.0:81->8080/tcp                  codex_edm_1
+566e59074f42   nginx:latest                   "/docker-entrypoint.…"   22 hours ago        Up About an hour             0.0.0.0:84->80/tcp                    codex_www_1
+7925cbde4337   wildme/wildbook-ia:nightly     "/docker-entrypoint.…"   22 hours ago        Up About an hour             0.0.0.0:82->5000/tcp                  codex_wbia_1
+77624c19e02d   postgres:10                    "docker-entrypoint.s…"   22 hours ago        Up About an hour             5432/tcp                              codex_db_1
+ba8962d4cf8e   gitlab/gitlab-ce:13.9.3-ce.0   "/assets/wrapper"        22 hours ago        Up About an hour (healthy)   22/tcp, 443/tcp, 0.0.0.0:85->80/tcp   codex_gitlab_1
+f259ede38b22   node:latest                    "/docker-entrypoint.…"   22 hours ago        Up About an hour                                                   codex_dev-frontend_1
+```
+
+These containers are available to enter on the command line using `docker exec -it [CONTAINER NAME] \bin\bash`. This command will grant you command line access as a root user for 
+whichever Codex application component you choose; `codex_dev-frontend_1` for the react front end, `codex_houston_1` for Houston or `codex_edm_1` for the EDM.
+
+Please refer to the Docker documentation for other common container actions.
+
+Development on Houston or other Codex components should be done by testing new code against the full application in these running docker containers. Running tests or migrations outside the docker-compose environment is an extra tool at your disposal but can be unpredictable and is not considered complete.
+
+### Mounted Directories
+
+#### Houston
+The docker-compose arrangement will attempt to mount local directories for development purposes. For Houston, this is the root repository directory. 
+
+If you create or modify a file in the local Houston repository you will be able to see the changes reflected when you `docker exec` inside the Houston container.
+
+This does not go the other way. Changes made inside the container reside only there.
```

Comment:
A quick test shows that this is not correct. 
Inside the container running `touch foo.bar `
causes foo.bar to appear in the local houston rep

Reply:
Yes, you should see the directory as the same inside the host and the container, like an nfs mounted directory, or a symlinked directory.  All changes are reflected in the container and host.